go/config/BUILD.bazel

```diff
@@ -58,6 +58,16 @@ string_list_flag(
     visibility = ["//visibility:public"],
 )
 
+string_flag(
+    name = "cover_format",
+    build_setting_default = "cover",
```

Comment:
Should we default to lcov so that coverage will work with the Bazel tooling by default?

Reply:
I would be very much in favor of this. It does break backwards compatibility in a minor way, but makes the feature more useful. I will push a new commit.

Reply:
If this is the consensus, I will squash the commits and update the commit message to reflect the new behavior after the review.